tensorflow/python/ops/parsing_ops.py

```diff
@@ -884,6 +884,24 @@ def decode_raw(input_bytes,
         input_bytes, out_type, little_endian=little_endian, name=name)
 
 
+@tf_export("io.encode_raw", v1=[])
+@dispatch.add_dispatch_support
+def encode_raw(tensor, name=None):
+  """Convert tensor into raw byte strings.
+
+  Args:
+    tensor:
+      The tensor to encode. The first dimension of the tensor will be
```

Comment:
The documentation should call out what tensor types are / are not supported. It seems like at least DT_RESOURCE and DT_VARIANT are not supported.

Reply:
Since the added test only tests numeric tests, the documentation should state that only numeric types are supported.

Reply:
@jsimsa Added.